offchain-modules/packages/x/src/handlers/ckb.ts

```diff
@@ -797,6 +1052,53 @@ export async function parseBurnTx(tx: Transaction): Promise<RecipientCellData |
   return cellData;
 }
 
+export function checkLock(amount: bigint, assetIdent: string, recipient: string, sudtExtraData: string): string {
+  const nervosAsset = ForceBridgeCore.config.eth.nervosAssetWhiteList.find(
+    (asset) => asset.typescriptHash === assetIdent,
+  );
+  if (!nervosAsset) {
+    return `nervos asset ${nervosAsset} not in nervos asset white list`;
+  }
+  const minimalAmount = nervosAsset.minimalBridgeAmount;
+  if (amount < BigInt(minimalAmount)) {
+    const humanizeMinimalAmount = new BigNumber(minimalAmount)
+      .times(new BigNumber(10).pow(-nervosAsset.decimal))
+      .toString();
+    return `minimal bridge amount is ${humanizeMinimalAmount} ${nervosAsset.symbol}`;
+  }
+  // check sudtSize
+  if (nervosAsset.typescriptHash !== '0x0000000000000000000000000000000000000000000000000000000000000000') {
```

Comment:
don't need to check sudtSize here, remove the check code

Reply:
removed the sudtSize check here